apis/autoscaling/v1alpha1/register.go

```diff
@@ -76,6 +76,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&PgpoolAutoscalerList{},
 		&PostgresAutoscaler{},
 		&PostgresAutoscalerList{},
+		&PgBouncerAutoscaler{},
+		&PgBouncerAutoscalerList{},
```

Comment:
Add all the other autoscaler types too.  seems, Many of them are missing.

Reply:
Nope.  Compare with this list : https://github.com/kubedb/apimachinery/tree/master/apis/autoscaling/v1alpha1